interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -628,6 +634,48 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  //-----------------------------------------------------------------------------
+  // The tests below are only work with gRpc interop test framework.
```

Comment:
This is unexpected. Why is this?

Maybe you mean it isn't a unit test, because the server must be a OAuth-supporting server?